.vscode/settings.json

```diff
@@ -1,4 +1,16 @@
 {
+  "files.eol": "\n",
+  "editor.insertSpaces": true,
+  "editor.indentSize": 2,
+  "files.encoding": "utf8",
```

Comment:
utf8 is probably the default (it's 2023 lol) maybe check which ones aren't needed bc they're already defaults?

Reply:
Sure, but better safe than sorry, IMO - it's always possible for someone to have adjusted their settings for some reason (maybe one of someone's own projects uses a different style), and we've had most of these in `.editorconfig` this whole time.

Are you sure we shouldn't explicitly specify settings that are already defaults?

Reply:
I'm not sure how VSCode handles user defaults vs project defaults tbh, but yeah, better safe than sorry I guess